### ISSUE TITLE
"GoogleAuthenticator"

### ISSUE THREAD

**Issue by michaeljohansen** [ilich/passport-2fa-totp#1]
Pretty sure there's an L missing here in "Google":

![image](https://user-images.githubusercontent.com/582580/50909082-601a5b00-142b-11e9-904d-3e48e32f0c67.png)

**Comment by getsnoopy** [comment]
Yes, this is a nasty bug as I just found out. Could someone fix this ASAP?

**Comment by Turbotailz** [comment]
I thought I was going crazy when I was trying to use destructuring assignment 😂
